html-report/src/components/SearchBar.js

```diff
@@ -1,21 +1,128 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import elasticlunr from 'elasticlunr';
 
+const SEARCH_FIELDS = ['package_name', 'class_name', 'name', 'id', 'status'];
+const SEARCH_REF = 'id';
+const EL_SEARCH = elasticlunr();
+const STATUSES = ['failed', 'ignored'];
```

Comment:
Well, maybe remove `Failed | Ignored` for now and add proper implementation later? :)

I mean it's not very obvious and may confuse people and they could even get used to it :D